server/persistence/mysql/games.go

```diff
@@ -39,7 +41,7 @@ const (
 		Phase TINYINT UNSIGNED,
 		CutCard SMALLINT,
 		Crib INT,
-		CurrentDealer VARCHAR(` + maxPlayerUUIDLenStr + `),
+		CurrentDealer VARCHAR(` + maxPlayerUUIDLenStr + `) CHARACTER SET utf8mb4 COLLATE utf8mb4_0900_as_cs,
```

Comment:
I found this collation [here](https://dev.mysql.com/doc/relnotes/mysql/8.0/en/news-8-0-1.html).